app.js

```diff
@@ -13,6 +13,14 @@ new CronJob(cronFrequency, function() {
   request.post('http://localhost/export-tasks');
 }, null, true);
 
+new CronJob('* * * * *', async () => {
```

Comment:
The cron pattern misses a field since in the 'cron' npm library seconds is the finest granularity (see https://www.npmjs.com/package/cron#available-cron-patterns)
I think the default should also be more in the range of 'each 10 minutes' instead of 'each minute', but if it's configurable, that's less important.